tests/system/src/test/java/test/javafx/scene/web/StraightLineTest.java

```diff
@@ -132,48 +135,47 @@ public void setupTestObjects() {
                     "<html>\n" +
                     "<head>\n" +
                     "<style>\n" +
+                    "body {\n" +
+                    "margin:0px;\n"+
+                    "}\n" +
                     "div {\n" +
                     "padding:0px;\n"+
-                    "  text-decoration: underline;\n" +
-                    "  text-decoration-thickness: 20px;\n" +
+                    "width:150px;\n"+
+                    "height:20px;\n"+
+                    "margin:0px;\n"+
+                    "text-decoration: underline;\n" +
+                    "text-decoration-thickness: 30px;\n" +
                     "font-size: 16px; \n" +
                     "}\n" +
                     "</style>\n" +
                     "</head>\n" +
                     "<body>\n" +
-                    "<div>TEST</dv>\n" +
+                    "<div>TEST UNDERLINE</div>\n" +
                     "</body>\n" +
                     "</html>");
-
         });
 
         assertTrue("Timeout when waiting for focus change ", Util.await(webViewStateLatch));
-        Util.sleep(1000);
 
         Util.runAndWait(() -> {
             WritableImage snapshot = straightLineTestApp.primaryStage.getScene().snapshot(null);
             PixelReader pr = snapshot.getPixelReader();
-
             int start_x = (int)webView.getEngine().executeScript("document.getElementsByTagName('div')[0].getBoundingClientRect().x");
             int start_y = (int)webView.getEngine().executeScript("document.getElementsByTagName('div')[0].getBoundingClientRect().y");
             int height = (int)webView.getEngine().executeScript("document.getElementsByTagName('div')[0].getBoundingClientRect().height");
             int width = (int)webView.getEngine().executeScript("document.getElementsByTagName('div')[0].getBoundingClientRect().width");
```

Comment:
Minor: I recommend switching `width` and `height` to match the order of `x` and `y`.